src/Image/configs.ts

```diff
@@ -0,0 +1,32 @@
+import { ElementPropTypes } from '@volusion/element-proptypes';
+
+export const defaultConfig = {
+    alt: '',
+    height: 270,
+    src: '',
+    title: '',
+    width: 360
```

Comment:
If we're setting the default width & height, should we set a default `src` value too?

Reply:
Makes sense.